### PR TITLE
Add/change ids for testing (EXPOSUREAPP-14120)

### PR DESCRIPTION
**Pull request by schauersbergern**
[Ticket](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14120)

IDs added/changed, screen numbers apply to attachment in ticket above:

Screen 1
trace_location_organizer_warn_info_fragment_title_id

Screen 2
trace_location_organizer_trace_locations_warn_list_fragment_title_id
warn_item_title
warn_item_address

Screen 3
trace_location_organizer_warn_duration_fragment_title_id
eventDescription
eventAddress
eventDate

Screen 4
trace_location_organizer_warn_tan_fragment_title_id

Screen 5
trace_location_organizer_category_fragment_title_id
header:
category_header
item:
category_item_title
category_item_subtitle

Screen 6
trace_location_create_fragment_title_id
trace_location_create_fragment_subtitle_id

Screen 7
trace_location_organizer_trace_locations_list_fragment_title_id
trace_locations_item_icon
trace_locations_item_description
trace_locations_item_address

Screen 8
cwa_logo
tl_organizer_title
tl_organizer_subtitle
qrCodeImage

### CODE REVIEW

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt [hunk 1]
```diff
@@ -76,8 +76,8 @@ class QrCodeDetailFragmentTest : BaseUITest() {
             ).toBundle()
         )
 
-        onView(withId(R.id.title)).check(matches(withText("Jahrestreffen der deutschen SAP Anwendergruppe")))
-        onView(withId(R.id.subtitle)).check(matches(withText("Hauptstr. 3, 69115 Heidelberg")))
+        onView(withId(R.id.tl_organizer_title)).check(matches(withText("Jahrestreffen der deutschen SAP Anwendergruppe")))
+        onView(withId(R.id.tl_organizer_subtitle)).check(matches(withText("Hauptstr. 3, 69115 Heidelberg")))
         onView(withId(R.id.eventDate)).check(matches(withText("21.06.2021, 18:00 - 21:00 Uhr")))
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt:81]: Maybe name these with the full name: `trace_location_organizer_title` to have consistencty.

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt [hunk 2]
```diff
@@ -76,8 +76,8 @@ class QrCodeDetailFragmentTest : BaseUITest() {
             ).toBundle()
         )
 
-        onView(withId(R.id.title)).check(matches(withText("Jahrestreffen der deutschen SAP Anwendergruppe")))
-        onView(withId(R.id.subtitle)).check(matches(withText("Hauptstr. 3, 69115 Heidelberg")))
+        onView(withId(R.id.tl_organizer_title)).check(matches(withText("Jahrestreffen der deutschen SAP Anwendergruppe")))
+        onView(withId(R.id.tl_organizer_subtitle)).check(matches(withText("Hauptstr. 3, 69115 Heidelberg")))
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt:80]: Maybe name these with the full name: `trace_location_organizer_title` to have consistencty.

File: Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml [hunk 3]
```diff
@@ -47,7 +48,7 @@
                     app:layout_collapseMode="parallax">
 
                     <TextView
-                        android:id="@+id/title"
+                        android:id="@+id/tl_organizer_title"
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml:51]: Here too, `trace_locations_organizer_title`

File: Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml [hunk 4]
```diff
@@ -61,7 +62,7 @@
                         tools:text="Jahrestreffen der deutschen SAP Anwendergruppe" />
 
                     <TextView
-                        android:id="@+id/subtitle"
+                        android:id="@+id/tl_organizer_subtitle"
```
Comment (SamuraiKek) [inline Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml:65]: And here

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt (hunk 2 above)
Comment (schauersbergern) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/eventregistration/organizer/QrCodeDetailFragmentTest.kt]: changed

File: Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml (hunk 3 above)
Comment (schauersbergern) [reply Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml]: changed

File: Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml (hunk 4 above)
Comment (schauersbergern) [reply Corona-Warn-App/src/main/res/layout/trace_location_organizer_qr_code_detail_fragment.xml]: changed

**Review by mtwalli — APPROVED** [review]

**Pull request merged**